pkg/function/kube_exec.go

```diff
@@ -96,8 +96,8 @@ func (kef *kubeExecFunc) Exec(ctx context.Context, tp param.TemplateParams, args
 	ctx = field.Context(ctx, consts.PodNameKey, pod)
 	_ = field.Context(ctx, consts.ContainerNameKey, container)
 	stdout, stderr, err := kube.Exec(cli, namespace, pod, container, cmd, nil)
-	format.Log(pod, container, stdout)
-	format.Log(pod, container, stderr)
+	format.LogWithCtx(ctx, pod, container, stdout)
```

Comment:
I feel like the pod name and container name might get logged twice here since we are adding those to the context above on lines 96/97. Could you please confirm this with a blueprint using `KubeExec` function?

Reply:
@pavannd1 you are right. I will update function to not log these fields for now. Later we can get rid of unused arguments.

Reply:
@pavannd1 I am fixing where this is the case.